### PR TITLE
Issuer setup guide

### PR DESCRIPTION
**Pull request by EbonyLouis**
## Deploy Preview Direct Link: 
- https://deploy-preview-1338--tbd-website-developer.netlify.app/docs/tbdex/issuer/vc-serverSetup?lang=Kotlin

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206859926400067

### CODE REVIEW

File: site/docs/tbdex/issuer/vc-serverSetup.mdx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  </div>
+</LanguageSwitchBlock>
+
+## Next Steps
+
+Now that you have your server environment set up, depending on your use case, you can proceed with one of the following guides:
+
+- [Credential Issuance Guide](/docs/tbdex/issuer/vc-issuance)
+
+- [Known Customer Credential Guide](/)
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-serverSetup.mdx:168]: we know the URL, so we can start adding it in. Will likely fail the build but that's cool. We know not to merge until the other doc is merged

```suggestion
- [Known Customer Credential Guide](/docs/tbdex/issuer/kcc/kcc-issuer/)
```

File: site/docs/tbdex/issuer/vc-issuance.mdx [hunk 2]
```diff
@@ -19,173 +19,9 @@ A Credential Issuance app operates as an API service, exposing endpoints for thi
...
-export { app as api };
-```
-
-</p>
-
-  </div>
-</LanguageSwitchBlock>
-
-With this basic structure, you can now design a credential and implement your issuance path.
+If you need to create a Credential Issuance app from scratch, check out the [Credential Issuance Server Setup Guide](/) for detailed instructions on the dependencies and packages needed to set up your server.
```
Comment (blackgirlbytes) [inline site/docs/tbdex/issuer/vc-issuance.mdx:24]: [Credential Issuance Server Setup Guide](/)

missing path

**Review by angiejones — APPROVED** [review]

File: site/docs/tbdex/issuer/vc-serverSetup.mdx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+```kt title=Application.kt
+fun main() {
+  embeddedServer(Netty, port = 8082) { // this can be any port you want to listen on
+      configureRouting()
+  }.start(wait = true)
+}
+```
+
+4. next, in the same file define your API routes:
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-serverSetup.mdx:114]:
```suggestion
4. In the same file, define your API routes:
```

File: site/docs/tbdex/issuer/vc-serverSetup.mdx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+```kt title=Application.kt
+fun Application.configureRouting() {
+        routing {
+            get("/example-route") {
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-serverSetup.mdx:119]: the routes can be anything, or are we suggesting specific routes?

File: site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/issuer/CredentialIssuanceTest.kt [hunk 5]
```diff
@@ -41,6 +45,34 @@ fun checkSanctionsList(): SanctionsListResult {
...
+    )
+
+    val options = CreateDidDhtOptions(
+        services = listOf(idvService),
+        publish = true 
+    )
+
+    val issuerBearerDid = DidDht.create(InMemoryKeyManager(), options)
+
+    // Update an existing DID with 'IDV' endpoint
```
Comment (angiejones) [inline site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/issuer/CredentialIssuanceTest.kt:64]: these should be 2 separate code snippets with instructions on when to use which

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by angiejones** [comment]
Note the build is failing due to linking to the Issuer guide that isn't yet published

File: site/docs/tbdex/issuer/vc-serverSetup.mdx (hunk 4 above)
Comment (EbonyLouis) [reply site/docs/tbdex/issuer/vc-serverSetup.mdx:119]: The routes can be anything they want them to be with the exception of the two `/.well-known/` endpoints and that's stated in both of the guides so I figured leaving the setup as example-routes was the best. Then while following along with the guide they can use the same route names or change them

**Pull request merged**